src/jit/valuenum.cpp

```diff
@@ -1135,13 +1135,30 @@ ValueNum ValueNumStore::VNForFunc(var_types typ, VNFunc func, ValueNum arg0VN, V
     }
 }
 
+//------------------------------------------------------------------------------
+// VNForMapStore : Evaluate VNF_MapStore with the given arguments.
+//
+//
+// Arguments:
+//    typ  -    Value type
+//    arg0VN  - Zeroth argument
```

Comment:
You should replace Zeroth, First and Second argument with a brief name or description.